### PR TITLE
chore: prep release v0.12.0

### PR DESCRIPTION
**Pull request by Panaetius**
Release 0.12.0

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -18,6 +18,110 @@
...
+-  **cli:** show lfs status of dataset files
+   (`#1575 <https://github.com/SwissDataScienceCenter/renku-python/issues/1575>`__)
+   (`a1c3e2a <https://github.com/SwissDataScienceCenter/renku-python/commit/a1c3e2af84d502f1e67288ea791ef2f910bc67af>`__)
+-  **cli:** verbose output for renku show
+   (`#1524 <https://github.com/SwissDataScienceCenter/renku-python/issues/1524>`__)
+   (`dae968c <https://github.com/SwissDataScienceCenter/renku-python/commit/dae968cdf0052c47603f0b8189ea24ee0b410d9a>`__)
+-  **core:** Adds renku dataset update for Zenodo and Dataverse
+   (`#1331 <https://github.com/SwissDataScienceCenter/renku-python/issues/1331>`__)
+   (`e38c51f <https://github.com/SwissDataScienceCenter/renku-python/commit/e38c51f084c2d7e59088de385f7eae035043f9c1>`__)
+-  **dataset:** immutability
```
Comment (mohammad-sdsc) [inline CHANGES.rst:111]: I am not sure if we should include this issue because it's not user-facing.

File: CHANGES.rst [hunk 2]
```diff
@@ -18,6 +18,110 @@
 Changes
 =======
 
+` <https://github.com/SwissDataScienceCenter/renku-python/compare/v0.11.6...v0.11.7>`__ (2020-11-03)
```
Comment (mohammad-sdsc) [inline CHANGES.rst:21]: We probably should mark this as `0.12.0` since we have a bit more than bug fixes 😉.

**Review by mohammad-sdsc — COMMENTED** [review]
Thanks! We should also update chart version: `helm-chart/renku-core/Chart.yaml`.

**Comment by Panaetius** [comment]
> Thanks! We should also update chart version: `helm-chart/renku-core/Chart.yaml`.

We already bumped it so the chart images have the right version

**Review by jsam — APPROVED** [review]
:rocket:

**Pull request merged**